### PR TITLE
DDF-04746 enable eslint recommended rules. rules we don't already fol…

### PR DESCRIPTION
**Pull request by rymach**
…low have been disabled as needed. this change is still beneficial because it ensures we don't break any new rules and provides a list of rule violations we should resolve

https://eslint.org/docs/rules/
https://github.com/yannickcr/eslint-plugin-react#recommended
note: no linting will occur until https://github.com/connexta/ace/pull/11 is merged and used by ddf

downstream projects are not impacted by these rules

#### Who is reviewing it? 
@gjvera 
@leo-sakh 
@mojogitoverhere 
@adimka 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@djblue

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #04746

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by rymach** [comment]
build now

File: ui/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    "extends": ["eslint:recommended", "plugin:react/recommended"],
+    rules: {
+        // TODO turn these recommended rules back on
```
Comment (mojogitoverhere) [inline ui/.eslintrc.js]: Should this be addressed here?

Comment (rymach) [reply ui/.eslintrc.js]: rules we don't already follow have been turned off. this TODO is a reminder to some day fix all the violations and turn the rules on

Comment (mojogitoverhere) [reply ui/.eslintrc.js]: I recommend updating the TODO with that info

**Comment by rymach** [comment]
build now

**Pull request merged**